src/MagnumPlugins/AssimpImporter/AssimpImporter.h

```diff
@@ -196,6 +196,7 @@ class MAGNUM_TRADE_ASSIMPIMPORTER_EXPORT AssimpImporter: public AbstractImporter
 
         MAGNUM_TRADE_ASSIMPIMPORTER_LOCAL bool doIsOpened() const override;
         MAGNUM_TRADE_ASSIMPIMPORTER_LOCAL void doOpenData(Containers::ArrayView<const char> data) override;
+        MAGNUM_TRADE_ASSIMPIMPORTER_LOCAL void doOpenState(const void* state, const std::string& filePath = {}) override;
```

Comment:
No need to have the `= {}` here, it's a private function that's never called from outside.

Reply:
One more thing: can you document what type should be passed to the `openState()` function in the "Access to internal importer state" section of the docs?

Besides that, I'm happy :) :+1: